bucket/meshlab.json

```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "http://www.meshlab.net/",
+    "description": "Open source system for processing and editing 3D triangular meshes.",
+    "version": "2016.12",
+    "license": "GPL-3.0-or-later",
```

Comment:
```suggestion
    "license": "GPL-3.0-only",
```

Reply:
I suppose it means `GPL-3.0-or-later`. (If I misunderstood it, please tell me)

![lic](https://user-images.githubusercontent.com/27724471/60379220-bc1c7e80-9a61-11e9-9ab1-2245a16950ac.png)

Reply:
Yeah, you are right :joy: